doc/interface/spatial_discretization_methods.rst

```diff
@@ -0,0 +1,127 @@
+.. spatial_discretization_methods:
+
+Spatial discretization methods
+==============================
+
+Group /input/model/unit_XXX/discretization/SPATIAL_METHOD - Details on the methods
+----------------------------------------------------------------------------------
+
+CADET offers two spatial discretization methods: Finite Volumes (FV) and Discontinuous Galerkin (DG).
+While both methods approximate the same solution to the same underlying model, they may differ in terms of computational performance.
```

Comment:
While both methods approximate the same solution to the [~same~] underlying model, they may differ in terms of computational performance.